tests/Test/Client/ContainsTest/ContainsTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace tests\RavenDB\Test\Client\ContainsTest;
+
+use RavenDB\Type\Collection;
+use RavenDB\Type\StringList;
+use tests\RavenDB\RemoteTestBase;
+use tests\RavenDB\Test\Client\ContainsTest\Entity\UserWithFavs;
+
+class ContainsTest extends RemoteTestBase
+{
+    public function testContainsTest(): void
+    {
+        $store = $this->getDocumentStore();
+        try {
+
+            $session = $store->openSession();
+            try {
+                $userCreator = function ($name, $favs) use ($session) {
+                    $user = new UserWithFavs();
+                    $user->setName($name);
+                    $user->setFavourites($favs);
+
+                    $session->store($user);
+                };
+
+                $userCreator("John", StringList::fromArray(["java", "c#"]));
```

Comment:
I think we don't need string lists - as this is build in java type. Can we use regular arrays here? array("java", "C"#) ?